src/libs/actions/CompanyCards.ts

```diff
@@ -121,19 +122,20 @@ function setWorkspaceCompanyCardFeedName(policyID: string, workspaceAccountID: n
     API.write(WRITE_COMMANDS.SET_COMPANY_CARD_FEED_NAME, parameters, onyxData);
 }
 
-function setWorkspaceCompanyCardTransactionLiability(workspaceAccountID: number, policyID: string, bankName: string, liabilityType: string) {
+function setWorkspaceCompanyCardTransactionLiability(workspaceAccountID: number, policyID: string, bankName: CompanyCardFeed, liabilityType: string) {
     const authToken = NetworkStore.getAuthToken();
+    const isCustomFeed = CardUtils.isCustomFeed(bankName);
+    const feedUpdates = {
+        [bankName]: {liabilityType},
+    };
```

Comment:
seems like you handled the updates three different ways on this PR :D could you please use just one, for example the one you have in here setWorkspaceCompanyCardTransactionLiability. ie you define `feedUpdates` and then

`settings: isCustomFeed ? {companyCards: feedUpdates} : {oAuthAccountDetails: feedUpdates},`

Reply:
Updated!